Src/Newtonsoft.Json/Bson/BsonReader.cs

```diff
@@ -175,7 +175,7 @@ private string ReadElement()
         }
 
         /// <summary>
-        /// Reads the next JSON token from the stream.
+        /// Reads the next JSON token from the underlying <see cref="BinaryReader"/> or <see cref="Stream"/>.
```

Comment:
I see. I thought it would make sense to have the comments in line with the public interface (one constructor accepting a `Stream`, another accepting a `BinaryReader`), but it indeed makes them somewhat unwieldy. Changed.